app/src/main/java/org/apache/taverna/mobile/data/local/PreferencesHelper.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.taverna.mobile.data.local;
+
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import org.apache.taverna.mobile.data.model.User;
+
+import rx.Observable;
+import rx.functions.Func0;
+
+public class PreferencesHelper {
```

Comment:
This is not good way make Preference Manager.
Here you have made method for saving user details rather than make method for saving String, int , Boolean value and pass the Key and Value that's the way of doing things.